jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/impl/PostgreSQLSqlDialect.java

```diff
@@ -345,7 +368,7 @@ public IdentifierCaseHandling getQuotedIdentifierHandling() {
 
     @Override
     public String applyQuote(final String identifier) {
-        return "\"" + identifier.replace("\"", "\"\"") + "\"";
+        return "\"" + identifier.toLowerCase().replace("\"", "\"\"") + "\"";
```

Comment:
Strictly speaking this line of code breaks the separation of concerns (SoC), especially in the light of the method name.
It is clear enough though.